benchmarks/kafka-snowflake/conduit/pipeline.yml

```diff
@@ -0,0 +1,56 @@
+version: "2.2"
+pipelines:
+  - id: kafka-to-snowflake
+    status: stopped
+    processors:
+      - id: parsingpayload
+        plugin: "json.decode"
+        settings:
+          field: .Payload.After
+      - id: fieldset
+        plugin: "field.set"
+        settings:
+          field: .Key
+          value: '{"id":"{{ .Payload.After.id }}"}'
+      - id: parsingkey
+        plugin: "json.decode"
+        settings:
+          field: .Key
```

Comment:
Too bad this can't be configured through the connector itself. Maybe we should file an issue for this.

Reply:
Actually, I noticed the connector does the unmarshalling itself ([here](https://github.com/conduitio-labs/conduit-connector-snowflake/blob/762f521b9ec170135d23d823dc25627c6e39dc61/destination/format/csv.go#L443-L487)). I just used processors without thinking of giving the processor a JSON tbh. I'm not sure it would have made a difference tbh, from what I see the connector is using the stdlib while the processor is using goccy/go-json, so the processor should theoretically even be faster.